WikiWatershed/model-my-watershed
# Add four new modificaton types.

Added modification tools for four more land use type: "chaparral", "desert", "tall grass prairie", and "short grass prairie".

~~There is still one land use type that is present in the micro app but not in the modification tools: "wetland".~~
I said that backwards: "wetland" is in the tools but not in the micro-app.

**To test**
   * Go to a scenario
   * Click on "Land Cover" to bring up a selection of draw tools
   * You should notice four new draw tools
   * If you draw polygons with the new tools, those polygons should be saved and restored as expected

Connects WikiWatershed/model-my-watershed#424
